.github/workflows/checks.yml

```diff
@@ -15,119 +19,138 @@ jobs:
           fetch-depth: 0
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@0.14.0
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.18.0
 
       - name: Check Version(s)
-        run: poetry run version-check version.py
-
-  build-documentation-job:
-    name: Build Documentation
-    needs: [version-check-job]
+        run: poetry run version-check `poetry run python -c "from noxconfig import PROJECT_CONFIG; print(PROJECT_CONFIG.version_file)"`
+          echo "Please enable the version check by replacing this output with shell command bellow:"
+          echo ""
+          echo "poetry run version-check <<VERSION_PY>>"
+          echo ""
+          echo "Note: <<VERSION_PY>> needs to point to the version file of the project (version.py)."
+          exit 1
```

Comment:
This is the template which you should replace. Either take what have been there before or as mentioned `poetry run version-check `poetry run python -c "from noxconfig import PROJECT_CONFIG; print(PROJECT_CONFIG.version_file)"` should work too.